index.js

```diff
@@ -1,9 +1,14 @@
 /*
-	MIT License http://www.opensource.org/licenses/mit-license.php
-	Author Tobias Koppers @sokra
+  MIT License http://www.opensource.org/licenses/mit-license.php
+  Author Tobias Koppers @sokra
 */
 module.exports = function(content) {
-	this.cacheable && this.cacheable();
-	this.value = content;
-	return "module.exports = " + JSON.stringify(content);
+  this.cacheable && this.cacheable();
+  this.value = content;
+
+  var stringified = JSON.stringify(content)
```

Comment:
`var stringified = ...` => `content = ...` and maybe rename `content` to `json`please :)

Reply:
the content is the raw text of the file.  It's not json

Reply:
Yep, agreed `src` as argument and `var json = JSON.stringify(src)` is better :)

Reply:
fixed